internal/scorecard/alpha/tests/basic_tests.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	scapiv1alpha2 "github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha2"
+)
+
+const (
+	BasicCheckStatusTest = "basic-check-status"
+	BasicCheckSpecTest   = "basic-check-spec"
+)
+
+// CheckStatusTest verifies that CRs have a status block
+func CheckStatusTest(conf TestBundle) scapiv1alpha2.ScorecardTestResult {
+	r := scapiv1alpha2.ScorecardTestResult{}
+	r.Name = BasicCheckStatusTest
+	r.Description = "Custom Resource has a Status Block"
```

Comment:
the description in the config.yaml is probably redundant since its included into the ScorecardTestResult by the test itself, I'll remove that in a future PR.  I suppose however, having it in the config file gives you the ability to customize the test description (e.g. Spanish translation).  The name in the config file serves as the entrypoint so those match what the image is expecting or else the test image will not know what test to run.

I think documenting image/entrypoint fields makes sense (option 2).   I was planning on doing the documentation in a separate PR/story currently.

I think the only way you can trust test results is to insure you are running a valid version of the test images or else you can spoof results, I'd consider that the domain of the pipeline/CI maintainers.

Reply:
> I think the only way you can trust test results is to insure you are running a valid version of the test images or else you can spoof results, I'd consider that the domain of the pipeline/CI maintainers.

But what is the method by with the pipeline/CI maintainers know which images are being used?  With the current design, the pipeline runs `operator-sdk scorecard --bundle <bundle>` and then the output shows the results. The image and entrypoint are never surfaced to the caller.

WDYT about having the config file contribute some of the fields to the result output and the test image log contributing the test result fields:

- config.yaml contributes:
  - image
  - entrypoint
  - labels
  - name
  - description

- test log contributes:
  - state (pass/fail/error)
  - errors
  - suggestions
  - log

So with this `--list` would output just fields from `config.yaml` and running the test would output fields from both.

Reply:
--list currently works off of the config file only, I was going to keep it that way....when the integration to the test image is coded, I think we'll need to decide to pull the name/description from either the config file or the image itself, not sure at this point.

Reply:
Sounds good. I can see arguments for name and description come from either config or the log output. What I'm after is that we're crystal clear how that works, and we probably need to document it once we have a decision if it isn't immediately obvious.